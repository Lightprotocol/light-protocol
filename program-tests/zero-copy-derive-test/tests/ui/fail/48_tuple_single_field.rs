// Edge case: Tuple struct with single field
use light_zero_copy_derive::{ZeroCopy, ZeroCopyMut};

#[derive(ZeroCopy, ZeroCopyMut)]
#[repr(C)]
pub struct SingleFieldTuple(pub u32);

fn main() {}
