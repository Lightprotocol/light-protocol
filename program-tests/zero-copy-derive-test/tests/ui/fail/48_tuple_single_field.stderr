error: ZeroCopy only supports structs with named fields
 --> tests/ui/fail/48_tuple_single_field.rs:6:28
  |
6 | pub struct SingleFieldTuple(pub u32);
  |                            ^^^^^^^^^
